policy/modules/contrib/frr.te

```diff
@@ -0,0 +1,108 @@
+policy_module(frr, 1.0.0)
+
+########################################
+#
+# Declarations
+#
+
+type frr_t;
+type frr_exec_t;
+init_daemon_domain(frr_t, frr_exec_t)
+
+type frr_log_t;
+logging_log_file(frr_log_t)
+
+type frr_tmp_t;
+files_tmp_file(frr_tmp_t)
+
+type frr_lock_t;
+files_lock_file(frr_lock_t)
+
+type frr_conf_t;
+files_config_file(frr_conf_t)
+
+type frr_unit_file_t;
+systemd_unit_file(frr_unit_file_t)
+
+type frr_var_run_t;
+files_pid_file(frr_var_run_t)
+
+########################################
+#
+# frr local policy
+#
+allow frr_t self:capability { dac_override dac_read_search chown kill net_bind_service net_raw setgid setuid };
+allow frr_t self:process { setcap setpgid };
+allow frr_t self:packet_socket create;
+allow frr_t self:netlink_route_socket rw_netlink_socket_perms;
+allow frr_t self:tcp_socket { connect connected_stream_socket_perms };
+allow frr_t self:udp_socket create_socket_perms;
+allow frr_t self:rawip_socket create_socket_perms;
+allow frr_t self:unix_stream_socket connectto;
```

Comment:
Try to sort alphabetically whenever possible.

Reply:
Fixed.